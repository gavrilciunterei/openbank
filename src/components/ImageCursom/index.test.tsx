import React from 'react';
import renderer from 'react-test-renderer';
import ImageCustom from './index';

it('Test ImageCustom', () => {
  const tree = renderer
    .create(<ImageCustom alt="test" src="./img.png" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
