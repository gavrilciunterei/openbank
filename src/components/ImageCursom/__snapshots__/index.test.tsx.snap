// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test ImageCustom 1`] = `
<img
  alt="test"
  className="sc-bdvvtL kbhvXq"
  src="./img.png"
/>
`;
